### ISSUE TITLE
hit-tolerance property in vl-interaction-select is not working as intended

### ISSUE THREAD

**Issue by betanets** [ghettovoice/vuelayers#250]
In my case I want to use this property to select a feature within some pixels, similar to behavior in this Openlayers guide: https://openlayers.org/en/latest/examples/hit-tolerance.html.

But I found out that it has no effect when I add this property to *vl-interaction-select* component.

Some investigations: 
1. I opened source code for *ol/interaction/Select.js* and changed *hitTolerance_* value to 90. After that I got the behavior that was required.
2. I opened source code for *vuelayers/lib/select-interaction/index.js* and added *hitTolerance: this.hitTolerance,* in *new SelectInteraction({...})* constructor. After that I also got the behavior that was required.

Example of my code is shown below:
```
<vl-map ref="map" data-projection="EPSG:4326" @mounted="onMapMounted" :move-tolerance="10">
      <vl-view ref="view" :enable-rotation="false"></vl-view>
      <vl-interaction-select :features.sync="selectedFeatures" :hit-tolerance="90">
            <template slot-scope="select">
            ...
           </template>
      </vl-interaction-select>
</vl-map>
...
```

If i'm wrong, could you explain how to use this property? I can't find any examples in an official docs.

---

Vuelayers 0.11.16, Vue 2.6.10.

**Comment by ghettovoice** [comment]
Hello @betanets ,
looks like I forgot to add it to the constructor call. 
I'll fix this soon.

Thanks for response!

**Comment by ghettovoice** [comment]
Fixed in v0.11.17
